partner_contact_gender/__openerp__.py

```diff
@@ -18,18 +17,22 @@
 
 {
     "name": "Contact gender",
-    "version": "8.0.1.0.0",
-    "category": "Customer Relationship Management",
-    "author": "Odoo Community Association (OCA), Grupo ESOC",
+    "summary": "Add gender field to contacts",
+    "version": "9.0.1.0.0",
+    "author": "Odoo Community Association (OCA)",
```

Comment:
Happy to add original back, but I would have thought if all authors were listed, that would make it very long and unweildy - I thought that was the purpose of contributors, to some degree...